envs/openmind.env

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+source ~/.projects
+
+# common neuroimaging tools
+module add openmind/c3d/1.0.0
+module add openmind/ants/2.1.0-3.8bed08
+module add openmind/freesurfer/6.0.0
+module add openmind/fsl/5.0.9
+module add openmind/afni/2016.03.08
+module add mit/matlab/2017b
+module add openmind/mricron/16.08.28
+module add openmind/sessionfinder/1.0.0
+
+# feel free to add any other available tools by:
+
+# type `module avail` to view all available modules
+# within this file, add `module add <MODULE_NAME>`
+
+export CONDA_ENVS_PATH=/om2/user/$(whoami)/miniconda
+source activate my_env
```

Comment:
My suggestions for `setup.sh` would break this functionality. If you decide to use the base environment and not create a new environment, remove this line.